src/components/data-exploration/CellInfo.jsx

```diff
@@ -1,19 +1,15 @@
 import React, { useState, useEffect } from 'react';
 import _ from 'lodash';
-import {
-  useSelector,
-} from 'react-redux';
 import { Card } from 'antd';
 import PropTypes from 'prop-types';
 
 const CellInfo = (props) => {
-  const { coordinates, componentType } = props;
+  const { coordinates, componentType, cellInfo } = props;
 
-  const cellInfo = useSelector((state) => state.cellInfo);
   const [cellInfoVisible, setCellInfoVisible] = useState(false);
 
   useEffect(() => {
-    if (!cellInfo.cellName) {
+    if (!cellInfo) {
       return;
     }
     if (cellInfo.componentType !== componentType) {
```

Comment:
Can you check if we need `componentType`? The value of `componentType` in `Embedding.jsx` is derived from `embedding.method`, which is the method of embedding chosen from configure embedding. The same value is also passed in `cellInfo`, so we don't seem to need this check

Reply:
nice catch!